driver-core/src/test/java/com/datastax/driver/core/SchemaChangesTest.java

```diff
@@ -482,8 +484,7 @@ public void should_notify_of_keyspace_drop(String keyspace) throws InterruptedEx
         }
         assert added != null;
         refreshSchema();
-        for (Metadata m : metadatas())
-            assertThat(m.getReplicas(keyspace, Bytes.fromHexString("0xCAFEBABE"))).isNotEmpty();
+        replicasPresentForKeyspace(keyspace).becomesTrue();
```

Comment:
Come to think of it, @adutra has addressed all of this in #666, so i'll wait for that to merge and rebase and take in his changes.